_test/variadic10.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"log"
+	"os"
+)
+
+func main() {
+	logger := log.New(os.Stdout, "test ", log.Lmsgprefix)
+	logger.Printf("args: %v %v", 1, "truc")
```

Comment:
just for testing, I've just tried this instead:

```
logger.Printf("args: %v %v %v", 1, "truc", "autre")
```

and it seems to be panicking.